doc-coverage/raco.rkt

```diff
@@ -0,0 +1,75 @@
+#lang racket
+
+(require raco/command-name
+         racket/cmdline
+         rackunit/docs-complete
+         "main.rkt")
+
+(module+ main
+
+  (define binding (make-parameter #f))
+  (define ratio (make-parameter #f))
+  (define ignore (make-parameter #f))
+  (define error-on-exit? (make-parameter #f))
+
+  (define args
+    (command-line
+     #:program (short-program+command-name)
+     #:once-any
+     [("-b" "--binding") b
+      "Check the documentation for a specific binding"
+      (binding (string->symbol b))]
+     [("-r" "--ratio") r
+      "Specify required documentation ratio"
+      (ratio (string->number r))]
+     [("-s" "--skip") s
+      "Specify regex of bindings to ignore"
+      (ignore (regexp s))]
+     #:args (file . files)
+     (cons file files)))
+
+  (for ([a (in-list args)])
+    (let/ec break
+
+      (with-handlers ([exn:fail? (lambda (e)
+                                   (set! a (string->symbol a)))])
```

Comment:
I would love to, although I'm not really sure how to. Basically, I want to see if the module as a string exists, and if not, see if the module as a symbol exists. If neither exist, then the module the user entered does not exist. The `namespace-require` is to bring the module into the namespace for the document checking functions.

I guess I could put it in a helper function if you think that would make it more clear for you. I'm generally opposed to making helper functions that are only used once in a linear algorithm like this though, as it needlessly makes the program more complicated. But the whole program is small enough I wouldn't mind.